### ISSUE TITLE
Drupal: Link alt text

### ISSUE THREAD

**Issue by rschwab** [UCSCLibrary/SiteImprove#41]
[Siteimprove link](https://my2.siteimprove.com/Accessibility/624596/77184199328/NextGen/Issue/1?conformance=&siteTargetIssueKinds=1,2&wcagVersion=21&pageSegments=&ruleName=sia-r69&ruleId=69&issueKind=1&tagFilterType=2&siteTarget.wcagVersion=21&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2)

**Comment by rschwab** [comment]
All from the news page, missing alt text on the image links. They've since been unpublished. Close after next crawl confirms this.
